panel/src/components/Forms/index.js

```diff
@@ -3,6 +3,7 @@ import Autocomplete from "./Autocomplete.vue";
 import Counter from "./Counter.vue";
 import Form from "./Form.vue";
 import FormButtons from "./FormButtons.vue";
+import Controls from "./Controls/Controls.vue";
```

Comment:
I suppose you have plans to add more Controls components to that folder, right?

Reply:
Yes, at least `<k-model-form-controls>`.